experimentalSetups/lego_setup_2motors_amc_amcbldc/hardware/mechanicals/setup-2motors_amc_amcbldc-mec.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE params PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<params xmlns:xi="http://www.w3.org/2001/XInclude" robot="setup" build="1">
+    <group name="GENERAL">
+        <param name="MotioncontrolVersion">  6                          </param>
+        <param name="Joints">                2                          </param>
+        <param name="AxisMap">               0          1               </param>
+        <param name="AxisName">         "setup_yaw"     "setup_roll"    </param>
+        <param name="AxisType">         "revolute"      "revolute"      </param>
+        <param name="Encoder">          182.044         182.044         </param>
+        <param name="fullscalePWM">     32000           32000           </param>
+        <param name="ampsToSensor">     1000.0          1000.0          </param>
+        <param name="Gearbox_M2J">     -196             -196            </param>    <!-- motor to joint -->
+        <param name="Gearbox_E2J">      1               1               </param>    <!-- encoder to joint -->
+        <param name="useMotorSpeedFbk"> 0               0               </param>
+        <param name="MotorType">        "BLL_MOOG"      "BBLL_MOOG"     </param>
+        <param name="Verbose">          0               0               </param>
+    </group>
+
+    <group name="LIMITS">
+        <param name="hardwareJntPosMax">     180    180         </param>
+        <param name="hardwareJntPosMin">     -180   -180        </param>
+        <param name="rotorPosMin">           0      0           </param>
+        <param name="rotorPosMax">           0      0           </param>
+    </group>
+
+    <group name="AMCBLDC">
+        <param name="HasHallSensor">         1      1           </param>
+        <param name="TemperatureSensorType"> NONE   NONE        </param>
```

Comment:
@marcoaccame yesterday we talked about this parameter, present in new yarp configuration guide but not supported now by the FW

Reply:
In [here](https://github.com/robotology/robots-configuration/blob/devel/iCubTemplates/iCubTemplateV6_0/hardware/mechanicals/body_part-ebX-jA_B-mec.xml#L52-L58) it is explained that `TemperatureSensorType = NONE` is equivalent to `HasTempSensor = 0` and that is better to use `TemperatureSensorType` because `HasTempSensor` will be dismissed.

And yes, we don't have yet any temperature sensor support in the motor we use and on `amc2c` or `amcbldc`.
So, pls use `TemperatureSensorType = NONE`.